app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventMapper.kt

```diff
@@ -62,6 +66,7 @@ class ProgramEventMapper @Inject constructor(
             },
             nameCategoryOptionCombo =
             getCategoryComboFromOptionCombo(event.attributeOptionCombo())?.displayName(),
+            displayOrgUnitName = dataSetOrgUnitNumber > 1,
```

Comment:
EventViewModel is a Domain model, indeed it is in `package org.dhis2.commons.data`, `displayOrgUnitName` is more related to presentation logic, so I will evaluate it in a UseCase as you are going to reuse it, and the inject the use case in the classes where you generate the UI model like `ListCardUiModel`. 
You can follow the [Android guidelines](https://developer.android.com/topic/architecture/domain-layer) to generate use cases.

Reply:
`displayOrgUnitName` moved to UI model